aioxmpp/avatar/__init__.py

```diff
@@ -50,8 +50,11 @@
 
 .. autoclass:: AvatarSet
 
-.. autoclass:: aioxmpp.avatar.service.AbstractAvatarDescriptor
+.. module:: aioxmpp.avatar.service
+.. currentmodule:: aioxmpp.avatar.service
+.. autoclass:: AbstractAvatarDescriptor
+   :members:
```

Comment:
Don’t use ``:members:``. Document the members explicitly using ``.. automethod`` et al. This allows for grouping the members according to their use-cases and adding explanatory text in-between, as well as excluding members which are not supposed to show up in the docs.

(on the downside, you can also easily forget to list a member ... :))